.travis.yml

```diff
@@ -35,7 +31,9 @@ before_install:
 install:
   # Customise the testing environment
   # ---------------------------------
+  - conda install --channel ioos pytest
   - conda install --channel ioos --file requirements.txt
-  - conda install --channel ioos nco pyncml pytest pynco
+  - conda install --channel ioos pyncml pynco
```

Comment:
Can you take the opportunity and change the channels here from `ioos` to `conda-forge`?